src/education.njk

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+nohero: true
+meta:
+    title: "FlowFuse for Education"
+    keywords: FlowFuse, FlowFuse for students, FlowFuse for educators, FlowFuse for educational institutions, FlowFuse for universities, FlowFuse for schools, FlowFuse for colleges, FlowFuse for learning, FlowFuse for teaching, FlowFuse for training, FlowFuse for research, FlowFuse for academic institutions, FlowFuse for educational organizations, FlowFuse for educational establishments, FlowFuse for educational bodies, FlowFuse for educational centers, FlowFuse for educational facilities
+description: "FlowFuse offers a comprehensive suite of tools and resources for educational institutions, students, and educators to accelerate learning, teaching, and research in Node-RED and industrial automation."
```

Comment:
@gstout52, I've added this meta information. Feel free to make any changes and commit them as you see fit.